Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -170,6 +170,8 @@ export function install(args: Arguments) : Result {
                         ...(args.privateCache ? [] : [
                             // We don't have a private cache so untrack the appdata cache folder
                             d`${Context.getMount("LocalAppData").path}/yarn`,
+                            // CB creates a junction from %LocalAppData%\yarn to d:\dbs\profile\bxlint\appdata\local\yarn
+                            d`d:/dbs/profile/bxlint/appdata/local/yarn`,

```

Comment:
Nora has a PR cleaning up all these directories for cloudbuild into a generated cloudbuild.config.dsc file.

Please sync with her if she can add this path there..